release_tester/arangodb/installers/tar.py

```diff
@@ -101,7 +101,7 @@ def install_package(self):
         logging.debug("package dir: {0.cfg.package_dir}- server_package: {0.server_package}".format(self))
 
         cmd = [self.tar,
-                   '-xzf',
```

Comment:
I don't think that this works. we deliver zip-compressed tar. 
Yes, this will only work with gnutar - if the system doesn't have this, `self.tar` needs to be set.

Reply:
It actually works.